src/components/editor/EditorArticleController.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+import _ from 'lodash';
+import EditAuthorList from './EditAuthorList';
+
+const categories = [
+  "Category 1",
+  "Category 2",
+  "Category 3"
+];
+
+export default class EditorArticleController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleSaveChanges = this.handleSaveChanges.bind(this);
+    this.mainFormChanged = this.mainFormChanged.bind(this);
+    this.handleMainFormChange = this.handleMainFormChange.bind(this);
+    this.authorListChanged = this.authorListChanged.bind(this);
+    this.safeSetState({
+      changed: "none",
+      saving: false
+    });
+  }
+
+  static getFalcorPath(params) {
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+  }
+
+  handleSaveChanges(event) {
+    event.preventDefault();
+  }
+
+  mainFormChanged(event) {
+    if (event.target.name === "addAuthor") {
```

Comment:
Better is that to set a onSubmit / onChange handle where this event is emitted, and stop it from percolating. Otherwise future people handling changeEvents between their hierachies might unexpectedly run in this.